WebApiCustomers/Controllers/CustomersController.cs

```diff
@@ -44,10 +44,10 @@ public IActionResult Post(Customer model)
         {
             return Created("", $"Customer created with Id: {model.Id}");
         }
-        return BadRequest("There is already a customer with this CPF and Email");
+        return BadRequest($"There is already a customer with this CPF: {model.Cpf} and Email: {model.Email}");
     }
 
-    [HttpPut]
+    [HttpPut("{id}")]
     public IActionResult Put(Customer model)
```

Comment:
```suggestion
    public IActionResult Put(long id, Customer model)
```
Se você não pegar o id que vem da rota não irá realizar a ação de atualizar, pois o id sempre vai ser 0 no body.